core/src/test/java/nl/nn/adapterframework/util/FilenameUtilsTest.java

```diff
@@ -0,0 +1,272 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * FilenameUtils Tester.
+ *
+ * @author <Sina Sen>
+ */
+public class FilenameUtilsTest {
+
+
+
+    /**
+     * Method: isSystemWindows()
+     */
+    @Test
+    public void testIsSystemWindows() throws Exception {
+        assertEquals(FilenameUtils.isSystemWindows(), true);
+    }
```

Comment:
You might run on windows, but many of our servers do not :) this will fail on Travis and Jenkins

Reply:
yes indeed, I removed this test case.